.github/workflows/build.yml

```diff
@@ -20,36 +20,8 @@ jobs:
         server-id: github # Value of the distributionManagement/repository/id field of the pom.xml
         settings-path: ${{ github.workspace }} # location for the settings.xml file
     - name: Build and Test with PostgreSQL
-      run: ./gradlew build -Pdocker -Pdb=pg
+      run: ./gradlew testAll -Pdocker
```

Comment:
@DavideD This is simpler in the pipeline file, but the major drawback is that it will now run everything in sequence . Before, everything ran in parallel (jobs run in parallel if they have no dependencies on other jobs).
I'd really like to see a solution that runs everything in parallel so that CI completes faster, but I don't think that will be possible with an all-in-one gradle task.

Reply:
This issue will become more pronounced as we add more DBs too

Reply:
My main concerned her is to keep everything aligned and not having to touch CI, if possible, everytime we add a new database. But I will have a look to see if I can find a different solution

Reply:
We can also skip the commit that update the pipeline for now. I still like to test everything locally from time.

Reply:
Since the CI jobs are defined as a plain file along with the rest of the source files in the repo "touching CI" is not as bad as if all these jobs had to be manually set up by someone logging into a Jenkins UI or something.
Perhaps one option is to use the matrix functions in github to generate the matrix of jobs (quarkus does this afaik) but currently dynamic matrixes are not supported by GH actions

Reply:
Let's keep it as it is for now and we might evolve it later if we need to. Soon I will have to create a JOB on our CI and we can revisit the issue there. I don't think there is much we can do about this specific issue with the current structure of the project.

Reply:
"our CI" meaning some internal Hibernate CI? Is there a requirement to be using that? The current github actions setup works pretty well IMO (and it's free!)